trucks.js

```diff
@@ -1,504 +1,634 @@
+    Twitter: 'https://twitter.com/fezonwheels',
   },
   {
     name: 'Flair Taco',
-    type: ['Mexican'],
-    schedule: ['Monday', 'Tuesday', 'Wednesday', 'Thursday', 'Friday', 'Saturday', 'Sunday'],
-    payment: ['Cash'],
+    type: ['Mexican',],
+    schedule: ['Monday', 'Tuesday', 'Wednesday', 'Thursday', 'Friday', 'Saturday', 'Sunday',],
+    payment: ['Cash',],
     description: 'Regular taco truck in the Fremont neighborhood - open 7 days a week and late night',
-    Facebook: 'https://www.facebook.com/pages/Flair-Taco/197832600252403'
+    Facebook: 'https://www.facebook.com/pages/Flair-Taco/197832600252403',
   },
   {
     name: 'Frank\'s Franks',
-    type: ['Hot Dogs'],
-    schedule: ['Monday', 'Sunday'],
-    payment: ['Cash', 'Cards'],
+    type: ['Hot Dogs',],
+    schedule: ['Monday', 'Sunday',],
+    payment: ['Cash', 'Cards',],
     description: 'Gourmet hot dogs, weiners, sausages, hot links and more, contact at 425.761.9447 or franksfranks3@gmail.com',
-    Facebook: 'https://www.facebook.com/franksfrankshotdogs'
+    Facebook: 'https://www.facebook.com/franksfrankshotdogs',
   },
   {
     name: 'Fruit Chatter Box',
-    type: ['Sweets'],
-    schedule: ['Sunday'],
-    payment: ['Cash', 'Cards'],
+    type: ['Sweets',],
+    schedule: ['Sunday',],
+    payment: ['Cash', 'Cards',],
     description: 'Fruit Chatter Box provides fresh, seasonal fresh fruit in cups or on a stick at local festivals and street fairs.',
     Facebook: 'https://www.facebook.com/fruitchatterbox',
-    Twitter: 'https://twitter.com/fruitchatterbox'
+    Twitter: 'https://twitter.com/fruitchatterbox',
   },
   {
     name: 'Full Tilt Ice Cream',
-    type: ['Sweets'],
-    payment: ['Cash', 'Cards'],
+    type: ['Sweets',],
+    payment: ['Cash', 'Cards',],
     description: 'Handmade ice creams!',
     website: 'http://www.fulltilticecream.com',
     Facebook: 'https://www.facebook.com/fulltilticecreams',
-    Twitter: 'https://twitter.com/fticecream'
-  }
-];
\ No newline at end of file
+    Twitter: 'https://twitter.com/fticecream',
+  },
+];
+
+// this module should support the following methods:
+// getTrucks() - return all trucks
+// getTruck(name) - return the truck object matching 'name'
+// getFoodTypes() - return unique list of all associated food types (underscore has a function to help)
+// filterByDay(day) - return trucks with 'day' in schedule (use your filterByDay function from Module 3 homework)
+// filterByFoodType(foodType) - return trucks with associated 'foodType'
+
+/* eslint-enable max-len */
+/* eslint-disable no-console */
+/* eslint-disable no-undef */
+module.exports = {
+
+  /* eslint-enable no-undef */
+
+  getTrucks: function () {
+    var trucks;
+    for (var i = 0; i < foodTrucks.length; i++) {
+      trucks += foodTrucks[i].name + ' :: ' + '\n';
+    }
+
+    return trucks;
+  },
+
+  getTruck: function (truck) {
+    var foundTruck = 0;
+    while (truck.charAt(0) === ':')
```

Comment:
Commented on this in Module_4 homework, but it bears repeating - always use braces (`{` and `}`) with conditional and looping blocks. Also, this code shouldn't be necessary as the truck name coming in shouldn't have a `:` in it.